share/spice/get_events/get_events.js

```diff
@@ -50,17 +50,29 @@
     function buildUrl(id) {
         return 'https://getevents.co/events/preview/'+id;
     }
+        
+    function checkFullDay(s, e) {
+        var dates = getStartEnd(s, e);
+        if ((dates.plain_start.format('H:mm')=='0:00') && (dates.plain_end.format('H:mm')=='23:59')) {
+            //console.log(dates.plain_start.format('H:mm'), dates.plain_end.format('H:mm'))
+            return true;
+        }
+        return false;
+    }
 
     function getStartEnd(s, e) {
-        var start = moment(s),
+        //var start = moment(s)
+        var start = moment(new Date(s)).utc(),
```

Comment:
I don't believe you need to create the date object here? Moment accepts strings and turns them into Date objects itself.